### ISSUE TITLE
Steam Deck lacking rumble

### ISSUE THREAD

**Issue by chanunnaki** [emoose/OutRun2006Tweaks#27]
Thanks for this amazing work guys!! This is like a dream come true.

I'm playing the game on my Steam deck and there doesn't appear to be any vibration... is this a known issue on Steam Deck? is there anything I can do to enable it?

**Comment by emoose** [comment]
Strange, saw some others mention it worked fine on deck for them: https://x.com/englebert3rd/status/1808910559389048886

You could try changing VibrationControllerId in the OutRun2006Tweaks.ini file and see if increasing it lets it work, maybe it's on a different device ID for some reason.

IIRC Steam also has some options to configure the controller too, not sure how that works with deck but maybe something there is stopping it?

**Comment by chanunnaki** [comment]
I’ve tried changing the ini to 1 and 2, but still nothing. I will dig a little deeper later and report back if i find anything. Maybe it’s an issue with my device.

**Comment by octopuserectus** [comment]
It works on mine, it's just very weak.

**Comment by AP-Hammerheart** [comment]
I'm having issues with no rumble on steam deck aswell. Tried changing controller id, but no luck so far.

> It works on mine, it's just very weak.

Are you using default settings?

**Comment by octopuserectus** [comment]
> Are you using default settings?
Yes, I haven't change anything controls-related. Place your finger on the touchpad and bump into some walls, it's barely noticeable with default Steam Deck settings.

**Comment by chanunnaki** [comment]
Ok, got around to testing this...

On OLED Deck, yes, there is vibration, but it's super, super weak... I actually thought it was just vibration from the speakers before, but if you put your thumbs on the thumbpads, yes, you can feel, but it's super weak.

I connected up Xbox Elite 2 controller and you have to reorder the controllers and make the Xbox pad controller No.2 for it to work, but the external pad has no rumble either. Changing the VibrationControllerId to 1 or 2 also does nothing.

Vibration of Deck and external controller works normally in other games, like Slipstream, for example.

**Comment by AP-Hammerheart** [comment]
> On OLED Deck, yes, there is vibration, but it's super, super weak... I actually thought it was just vibration from the speakers before, but if you put your thumbs on the thumbpads, yes, you can feel, but it's super weak.

I'm using the LCD deck and there is no vibration. Tried a few settings but so far, no dice.
